subsys/net/lib/app/init.c

```diff
@@ -90,12 +90,12 @@ static void setup_dhcpv4(struct net_if *iface)
 #define setup_dhcpv4(...)
 #endif /* CONFIG_NET_DHCPV4 */
 
-#if defined(CONFIG_NET_IPV4) && !defined(CONFIG_NET_DHCPV4)
-
-#if !defined(CONFIG_NET_APP_MY_IPV4_ADDR)
+#if defined(CONFIG_NET_IPV4) && !defined(CONFIG_NET_DHCPV4) && !defined(CONFIG_NET_APP_MY_IPV4_ADDR)
```

Comment:
You should be >80 with this line

Reply:
Fixed, used "\" for line continuation.

Reply:
Was not a blocking comment, but thanks for taking it anyway!